go/enclave/db/rawdb/accessors_chain.go

```diff
@@ -175,10 +175,14 @@ func readBatchBodyRLP(db ethdb.KeyValueReader, hash common.L2RootHash) (rlp.RawV
 	return data, nil
 }
 
-func WriteL2HeadBatch(db ethdb.KeyValueWriter, l1Head common.L1RootHash, l2Head common.L2RootHash) error {
```

Comment:
Before the updating of the head and the L1->L2 mapping was done at the same time. I've moved this into separate methods so we can set the head batch when we need to (but you can pass in the dbBatch so it can still be atomic).